<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>526</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="styleSheet">
       <string notr="true">QStackedWidget {
margin-right: 15px; 
margin-left: 8px;
}</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:24pt; font-weight:700; color:#2a2a2a;&quot;&gt;Đăng nhập&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
            <property name="textFormat">
             <enum>Qt::AutoText</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_3">
            <property name="styleSheet">
             <string notr="true">QLabel {
margin-bottom: 25px;
}
</string>
            </property>
            <property name="text">
             <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; color:#717171;&quot;&gt;Sử dụng tài khoản FPT Camera của bạn&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout">
            <item>
             <widget class="customWidget" name="widget_number" native="true">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="styleSheet">
               <string notr="true">#widget_number{
border: 2px solid rgb(173, 173, 173);
border-radius: 5px;
background-color: rgb(180, 180, 180);
}

#widget_number:hover {
border: 2px solid rgb(139, 139, 139);
}
</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI Emoji</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">QLabel {
border: 0px;
height: 32px;
background-color: rgb(180, 180, 180);
}</string>
                 </property>
                 <property name="text">
                  <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;+84&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="customLineEdit" name="lineEdit_number">
                 <property name="font">
                  <font>
                   <family>Segoe UI Emoji</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#lineEdit_number {
border: 0px;
height: 32px;
color: rgb(43, 43, 43);
padding-left:  5px;
padding-right: 20px;
background-color: rgb(180, 180, 180);
}</string>
                 </property>
                 <property name="inputMask">
                  <string/>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>10</number>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Normal</enum>
                 </property>
                 <property name="dragEnabled">
                  <bool>false</bool>
                 </property>
                 <property name="placeholderText">
                  <string>Số điện thoại của bạn</string>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <widget class="customWidget" name="widget_pass" native="true">
              <property name="styleSheet">
               <string notr="true">#widget_pass{
border: 2px solid rgb(173, 173, 173);
border-radius: 5px;
background-color: rgb(180, 180, 180);
}

#widget_pass:hover {
border: 2px solid rgb(139, 139, 139);
}



</string>
              </property>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="customLineEdit" name="lineEdit_pass">
                 <property name="enabled">
                  <bool>true</bool>
                 </property>
                 <property name="font">
                  <font>
                   <family>Segoe UI Emoji</family>
                   <pointsize>11</pointsize>
                  </font>
                 </property>
                 <property name="focusPolicy">
                  <enum>Qt::StrongFocus</enum>
                 </property>
                 <property name="styleSheet">
                  <string notr="true">#lineEdit_pass {
border: 0px;
height: 32px;
color: rgb(43, 43, 43);
padding-left:  20px;
padding-right: 20px;
background-color: rgb(180, 180, 180);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                 <property name="maxLength">
                  <number>25</number>
                 </property>
                 <property name="echoMode">
                  <enum>QLineEdit::Password</enum>
                 </property>
                 <property name="placeholderText">
                  <string>Mật khẩu của bạn</string>
                 </property>
                 <property name="cursorMoveStyle">
                  <enum>Qt::LogicalMoveStyle</enum>
                 </property>
                 <property name="clearButtonEnabled">
                  <bool>false</bool>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QPushButton" name="button_pass">
                 <property name="styleSheet">
                  <string notr="true">QPushButton {
border: 0px;
background-color: rgb(180, 180, 180);
}</string>
                 </property>
                 <property name="text">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QCheckBox" name="checkBox">
                <property name="font">
                 <font>
                  <italic>false</italic>
                  <bold>false</bold>
                 </font>
                </property>
                <property name="styleSheet">
                 <string notr="true">#checkBox {
color: #717171;
}</string>
                </property>
                <property name="text">
                 <string>Giữ trạng thái đăng nhâp</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="text">
                 <string>PushButton</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_2">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_3">
              <property name="text">
               <string>PushButton</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="page_2"/>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label_img">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
      <property name="text">
       <string>image</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customWidget</class>
   <extends>QWidget</extends>
   <header>customwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>customLineEdit</class>
   <extends>QLineEdit</extends>
   <header>customlineedit.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
